examples/stm32f4_discovery/display/nokia_5110/project.cfg

```diff
@@ -0,0 +1,27 @@
+
+[general]
+name = n5110
+
+[build]
+device = stm32f407vg
+clock = 168000000
+buildpath = ${xpccpath}/build/stm32f4_discovery/display/${name}
+
+[program]
+tool = openocd
+
+[defines]
+CORTEX_VECTORS_RAM = 0
```

Comment:
This is copy-paste from other examples. See 86baf1f for the removal in other examples